tslearn/docs/clustering.rst

```diff
@@ -0,0 +1,116 @@
+.. _clustering:
+
+Time Series Clustering
+======================
+
+Dedicated methods exist for time series clustering.
+A first thing to understand is why there is a need for dedicated methods.
+
+The following Figure illustrates this need (code to reproduce is available
+:ref:`in the Gallery of Examples
+<sphx_glr_auto_examples_clustering_plot_kmeans.py>`):
+
+.. figure:: _static/img/kmeans.svg
+    :width: 100%
+    :align: center
+
+    :math:`k`-means clustering with Euclidean distance. Each subfigure represents series from a given cluster and their centroid (in red).
+
+This Figure is the result of a :math:`k`-means clustering that uses Euclidean
+distance as a base metric.
+One issue with this metric is that it is not invariant to time shifts, while
+the dataset at stake clearly holds such invariants.
+
+:math:`k`-means and Dynamic Time Warping
+----------------------------------------
+
+Hence, it would be profitable to use :math:`k`-means clustering with a metric
```

Comment:
To overcome the previously illustrated issue, distance metrics dedicated to time series, such as DTW, are required. As can be seen in the Figure below, the usage of such metrics produce more meaningful results.